batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,215 @@
+        String urlPath = request.getPathInfo();
+
+        if (urlPath == null || urlPath.equals("/")) {
+            int page = 1;
+
+            if (request.getParameter("page") != null) {
+                try {
+                    page = Integer.parseInt(request.getParameter("page"));
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();
+                    response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+                    request.setAttribute("message", e.getMessage());
+
+                    RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+                    view.forward(request, response);
+                }
+            } else {
+                page = 1;
+            }
+            list(request, response, page);
+
+        } else {
+            String[] paths = urlPath.split("/");
+            System.out.println(Arrays.toString(paths));
+            if (paths[1].equals("create")) {
+                create(request, response);
+            } else if (paths[2].equals("edit")) {
+                try {
+                    int id = Integer.parseInt(paths[1]);
+                    edit(request, response, id);
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();
+                    response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+                    request.setAttribute("message", e.getMessage());
+
+                    RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+                    view.forward(request, response);
+                }
+            }
+        }
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String urlPath = request.getPathInfo();
+
+        if(urlPath != null || !urlPath.equals("/")){
+            String[] paths = urlPath.split("/");
+            if (paths[1].equals("create")) {
+                createProcess(request, response);
+            } else if (paths[2].equals("edit")) {
+                try {
+                    int id = Integer.parseInt(paths[1]);
+                    editProcess(request, response, id);
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();
+                    response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+                    request.setAttribute("message", e.getMessage());
+
+                    RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/views/error.jsp");
+                    view.forward(request, response);
+                }
+            } else if (paths[2].equals("delete")) {
+                try {
+                    int id = Integer.parseInt(paths[1]);
+                    deleteProcess(request, response, id);
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)